src/timg.cc

```diff
@@ -224,6 +226,9 @@ int main(int argc, char *argv[]) {
         case 'c':
             loops = atoi(optarg);
             break;
+        case 'C':
```

Comment:
I think we should make the clear option a bit smarter:
  - whenever we have an animation (so more than one frame) or display animation, Clear is assumed by default and it can not be switched off (as we otherwise would get a wierd scrolling).
  - Whenever we have exactly one image to display, by default we don't clear the screen but just display it there. Unless the user choose `-C` which does the screen clearing.
So here, we should just record if `clear_requested` from the flag.

So after you have gathered the flag here, you need to do something below
  const bool needs_cleaning = do_scroll || frames.size() > 1 || clear_requested;
.. and use that in the function calls and constructors.